primitives/src/lib.rs

```diff
@@ -60,10 +60,17 @@ pub const DEFAULT_UNIT_CREATION_DELAY: u64 = 300;
 
 pub const DEFAULT_COMMITTEE_SIZE: u32 = 4;
 
-pub const DEFAULT_KICK_OUT_MINIMAL_EXPECTED_PERFORMANCE: Perbill = Perbill::from_percent(0);
-pub const DEFAULT_KICK_OUT_SESSION_COUNT_THRESHOLD: SessionCount = 3;
-pub const DEFAULT_KICK_OUT_REASON_LENGTH: u32 = 300;
+pub const DEFAULT_BAN_MINIMAL_EXPECTED_PERFORMANCE: Perbill = Perbill::from_percent(0);
+pub const DEFAULT_BAN_SESSION_COUNT_THRESHOLD: SessionCount = 3;
+pub const DEFAULT_BAN_REASON_LENGTH: u32 = 300;
 pub const DEFAULT_CLEAN_SESSION_COUNTER_DELAY: SessionCount = 960;
+pub const DEFAULT_BAN_PERIOD: EraIndex = 10;
+
+#[derive(Decode, Encode, TypeInfo, Debug, Clone)]
+pub enum ElectionOpenness {
```

Comment:
Maybe `ElectionsMode` is less of a mouthful?

Reply:
It's more ambiguous imo, let's wait for an opinion from the second reviewer :)